<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="init">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Initialiser</string>
     </property>
    </widget>
    <widget class="QPushButton" name="param">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>11</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Préférences</string>
     </property>
    </widget>
    <widget class="QPushButton" name="jouer">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Jouer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QLabel" name="aide1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>35</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="aide2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>35</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="aide4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>35</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QPushButton" name="load">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>335</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>200</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Vider</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>375</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retirer">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>160</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Retirer</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="partition">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>280</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Partition</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="duplicate">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QLabel" name="aide3">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>358</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="aide3_2">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>303</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insert">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>100</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Insérer</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>120</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Informations</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ports :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Connectés :</string>
     </property>
    </widget>
    <widget class="QLabel" name="nbremoteur">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="nbreconnecte">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PAS PRÊT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Homing :</string>
     </property>
    </widget>
    <widget class="QLabel" name="homing">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>150</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>false</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="seq">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>461</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Séquence</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuS_quences">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionCharger"/>
    <addaction name="actionSauvegarder"/>
   </widget>
   <widget class="QMenu" name="menuPartition">
    <property name="title">
     <string>Partition</string>
    </property>
    <addaction name="actionCharger_2"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionPremi_re_fois"/>
    <addaction name="actionCacher_l_aide"/>
   </widget>
   <addaction name="menuS_quences"/>
   <addaction name="menuPartition"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCharger_une_s_quence">
   <property name="text">
    <string>Charger une séquence</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionCharger_2">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
  <action name="actionPremi_re_fois">
   <property name="text">
    <string>Première fois ?</string>
   </property>
  </action>
  <action name="actionCacher_l_aide">
   <property name="text">
    <string>Cacher l'aide</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
